docs/basics/sso/setting-up-sso.md

```diff
@@ -20,14 +20,24 @@ Sauce Labs supports Identity Provider (IdP)-initiated and Service Provider (SP)-
 First, set up a new SAML application in your [Identity Provider (IdP)](#setting-up-identity-provider). Next, enable and configure SSO in [Sauce Labs Team Management Panel](#integrating-with-sauce-labs-service-provider).
 
 :::note
-SSO at Sauce Labs is global. In other words, it does not matter in which region 
+SSO at Sauce Labs is global. In other words, it does not matter in which region
 you set it up. Unlike to the [old SSO implementation](/basics/sso-deprecated/setting-up-single-sign-on), which has been deprecated.
 :::
 
 ## Setting Up Identity Provider
 
+:::tip
+We provide preconfigured SAML applications for some identity providers. They allow you to set up SSO integration with just a few clicks. Choose your provider and follow the configuration tutorial:
+* [OneLogin](/basics/sso/configuring-sso-in-onelogin)
+:::
+
+If you use a custom identity provider or your provider does not appear in the list above, please follow the tutorial for manual setup:
+
 1. Obtain SAML metadata from Sauce Labs Service Provider, which is served under [this link](https://accounts.saucelabs.com/am/sso/metadata/https%3A%2F%2Faccounts.saucelabs.com%2Fsp).
-2. Log in to your IdP admin account, such as Okta, OneLogin, or Microsoft Azure Active Directory, etc.
+2. Log in to your identity provider administrator panel.
+   1. For example in Okta, you log into your account and next click **Admin** button.
+   <img src={useBaseUrl('img/basics/sso/setup-sso-idp-admin-login.png')} alt="IdP Admin Login" width="900" />
+   
 3. Create a new SAML application.
    - In the IdP, go to the "Applications" or "Integrations" section and create a new SAML application.
    - Follow the instructions from your IdP to provide the Sauce Labs SAML metadata obtained in step 1. If your IdP does not allow uploading service provider metadata, set up the integration manually using the information provided [in the paragraphs below](#service-provider-saml-requirements).
```

Comment:
```suggestion
   - Follow the instructions from your IdP to provide the Sauce Labs SAML metadata obtained in Step 1. If your IdP does not allow uploading service provider metadata, set up the integration manually using the information provided [in the paragraphs below](#service-provider-saml-requirements).
```

Reply:
fxied

Reply:
I'm not sure if the suggested change is right, I changed it to:
```
set up the integration manually using [Service Provider SAML Requirements](#service-provider-saml-requirements).
```